test/testenv/secretutil.go

```diff
@@ -160,9 +167,9 @@ func GetSecretDataMap(hecToken string, password string, pass4SymmKey string, idx
 	return updatedSecretData
 }
 
-// CheckAdminPasswordViaAPI check if password can be used to access api
-func CheckAdminPasswordViaAPI(deployment *Deployment, podName string, password string) bool {
-	cmd := fmt.Sprintf("curl -iks -u admin:%s https://localhost:8089/servicesNS/admin/splunk_httpinput/data/inputs/http?output_mode=json", password)
+// CheckSecretViaAPI check if secret (hec token or password) can be used to access api
+func CheckSecretViaAPI(deployment *Deployment, podName string, secretName string, secret string) bool {
+	cmd := fmt.Sprintf(SecretAPIRequests[secretName], secret)
 	output, err := ExecuteCommandOnPod(deployment, podName, cmd)
 	if err != nil {
 		return false
```

Comment:
Can we capture all response codes as string maps or just const strings? i.e **"HTTP/1.1 200 OK"**